template/template.go

```diff
@@ -76,3 +84,137 @@ func (t *Template) ExecuteHTMLString(html string, data interface{}) (string, err
 	err = tmpl.Execute(&buf, data)
 	return buf.String(), err
 }
+
+type FuncMap map[string]interface{}
+
+var DefaultFuncs = FuncMap{
+	"upper": strings.ToUpper,
+	"lower": strings.ToLower,
+	"title": strings.Title,
+	// pairs allows for in-order iteration of key/value pairs.
+	"pairs": func(m map[string]string) []Pair {
+		var (
+			pairs     = make([]Pair, 0, len(m))
+			keys      = make([]string, 0, len(m))
+			sortStart = 0
+		)
+		for k := range m {
+			if k == string(model.AlertNameLabel) {
```

Comment:
Yes, this would be a very rare edge case with little impact. Worth it for the gain of simplicity.